kernel/ramfs.c

```diff
@@ -916,8 +916,13 @@ static int _fs_open(
             ramfs, locals->dirname, true, NULL, &parent, NULL, NULL));
 
         /* Create the new file inode */
-        ECHECK(_inode_new(
-            ramfs, parent, locals->basename, (S_IFREG | mode), &inode));
+        /* ATTN: Current ramfs only support S_IFREG or S_ISCHR (virtual /dev) */
```

Comment:
Are we allowing creating virtual files with `open`?

Reply:
Yes. `myst_create_virtual_file()` use `fs_open()`